### ISSUE TITLE
Types not exported, causing issues with NodeNext

### ISSUE THREAD

**Issue by cevr** [CyanSalt/vite-plugin-legacy-swc#4]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `vite-plugin-legacy-swc@0.4.0` for the project I'm working on.

It seems like the "types" export is missing, causing issues with typescript when using `moduleResolution: "NodeNext"`

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/vite-plugin-legacy-swc/package.json b/node_modules/vite-plugin-legacy-swc/package.json
index 62cbde3..f35c247 100644
--- a/node_modules/vite-plugin-legacy-swc/package.json
+++ b/node_modules/vite-plugin-legacy-swc/package.json
@@ -8,6 +8,7 @@
   "types": "./dist/index.d.ts",
   "exports": {
     ".": {
+      "types": "./dist/index.d.ts",
       "import": "./dist/index.mjs",
       "require": "./dist/index.cjs"
     }
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by CyanSalt** [comment]
You are right. Feedback is greatly appreciated and I will fix it.

**Comment by CyanSalt** [comment]
Fixed in [v0.4.1](https://github.com/CyanSalt/vite-plugin-legacy-swc/releases/tag/v0.4.1).
